pkg/README.md

```diff
@@ -4,7 +4,7 @@
 -   [Overview](#overview)
 -   [Roadmap](#roadmap)
 -   [Reference](docs/reference.md)
```

Comment:
Should this start with a slash as well?

Reply:
I could, but it would be /pkg/docs/reference.md.  docs is a sibling of this file, examples is a level up